functions/date.ts

```diff
@@ -25,10 +26,10 @@ export function calculateAgeInYears(birthDate: Date, otherDate: Date = new Date(
 // Returns the next birthday after the given date
 export function calculateNextBirthday(birthDate: Date, today: Date): Date {
   const nextBirthDay = new Date()
```

Comment:
We should use `Date.UTC(year, monthIndex, day)` here. 

See also: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/UTC